### PR TITLE
Remove interactive docker exec calls to allow use in unattended scripts.

### PR DESCRIPTION
**Pull request by haiqi96**
# Description
Remove the "-it" from the docker command in the start CLP script because cloud-init doesn't support docker with interactive mode

# Validation performed
Ran the package locally and on the docker and verifed that the script waits for database and queue to start.

### CODE REVIEW

**Review by kirkrodrigues — CHANGES_REQUESTED** [review]

File: components/clp-package-utils/clp_package_utils/scripts/start_clp.py [hunk 1]
```diff
@@ -270,12 +273,11 @@ def start_queue(instance_id: str, clp_config: CLPConfig):
 
     # Wait for queue to start up
     # fmt: off
-    cmd = [
-        "docker", "exec", "-it", container_name,
-        "rabbitmqctl", "wait", str(rabbitmq_pid_file_path),
-    ]
+    rabbitmq_cmd = ["rabbitmqctl", "status"]
```
[the remark this answers is not in the document]
Comment (haiqi96) [reply components/clp-package-utils/clp_package_utils/scripts/start_clp.py]: You are right, the [rabbitmq-diagnostics check_running](https://www.rabbitmq.com/rabbitmq-diagnostics.8.html#check_running) seems to be what I should really use.

File: components/clp-package-utils/clp_package_utils/scripts/start_clp.py [hunk 2]
```diff
@@ -71,30 +71,23 @@ def append_docker_port_settings_for_host_ips(
...
+            subprocess.run(cmd, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL, check=True)
             return True
         except subprocess.CalledProcessError:
             if time.time() - begin_time > timeout:
                 break
             time.sleep(1)
 
-    logger.error(f"Timeout while waiting for {DB_COMPONENT_NAME} to initialize.")
+    cmd_str = " ".join(cmd_to_run)
+    logger.error(f"Timeout while waiting for cmd {flatten_cmd} to run after {timeout} seconds")
```
Comment (kirkrodrigues) [inline components/clp-package-utils/clp_package_utils/scripts/start_clp.py:90]:
```suggestion
    logger.error(f"Timeout while waiting for command '{cmd_str}' to run after {timeout} seconds")
```

Comment (kirkrodrigues) [reply components/clp-package-utils/clp_package_utils/scripts/start_clp.py:90]: Missed from last review

File: components/clp-package-utils/clp_package_utils/scripts/start_clp.py [hunk 3]
```diff
@@ -269,13 +272,10 @@ def start_queue(instance_id: str, clp_config: CLPConfig):
...
     # Wait for queue to start up
-    # fmt: off
-    cmd = [
-        "docker", "exec", "-it", container_name,
-        "rabbitmqctl", "wait", str(rabbitmq_pid_file_path),
-    ]
-    # fmt: on
-    subprocess.run(cmd, stdout=subprocess.DEVNULL, check=True)
+    rabbitmq_cmd = ["rabbitmq-diagnostics", "check_running"]
+
```
Comment (kirkrodrigues) [inline components/clp-package-utils/clp_package_utils/scripts/start_clp.py:276]:
```suggestion
```

**Review by kirkrodrigues — CHANGES_REQUESTED** [review]

File: components/clp-package-utils/clp_package_utils/scripts/start_clp.py (hunk 2 above)
Comment (haiqi96) [reply components/clp-package-utils/clp_package_utils/scripts/start_clp.py:90]: GG

**Review by kirkrodrigues — APPROVED** [review]
Commit msg: `Remove interactive docker exec calls to allow use in unattended scripts. (#282)`?

**Pull request merged**